avaArzl/README.md

```diff
@@ -0,0 +1,57 @@
+# avaArzl 
+
+## Event 201901
+
+### Changelog 
+#### V1 -> V2
+- forest.shp
+- postprocessed raw data from Waldkategorienlayer
+- converted multipolygon to single polygon
+- removed very small features and holes were cut open
```

Comment:
only the changes from V1-V2 - so here it would just be that the forest streets are closed right?